### PR TITLE
feat: http endpoint test support

### PR DESCRIPTION
**Pull request by drduhe**
*Issue #, if available:*

*Description of changes:*
* Adding support for testing HTTP model endpoints
* ``process_image.py`` now takes an endpoint type as an arg - defaults to sm

*Testing:*
* Ran all integration tests against up to date deployment of OSML.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/aws/osml/integ/http_centerpoint/test_http_centerpoint_model.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        s3_client=s3_client(),
+        kinesis_client=kinesis_client(),
+    )
+
+    # validate the number of region request that were created in the process and check if they are succeeded
+    region_request_count = count_region_request_items(image_id=image_id, ddb_client=ddb_client())
+    validate_expected_region_request_items(region_request_count)
+
+
+def get_load_balancer_dns_url(load_balancer_name: str) -> str:
```
Comment (RanbirAulakh) [inline src/aws/osml/integ/http_centerpoint/test_http_centerpoint_model.py:59]: Missing docstrings

File: src/aws/osml/utils/osml_config.py [hunk 2]
```diff
@@ -26,6 +26,11 @@ class OSMLConfig:
     SM_FLOOD_MODEL: str = os.getenv("SM_FLOOD_MODEL", "flood")
     SM_AIRCRAFT_MODEL: str = os.getenv("SM_AIRCRAFT_MODEL", "aircraft")
 
+    # HTTP model config
+    HTTP_CENTERPOINT_MODEL_URL: str = os.getenv("HTTP_CENTER_POINT_MODEL_URL", None)
+    HTTP_CENTERPOINT_MODEL_ELB_NAME: str = os.getenv("HTTP_CENTER_POINT_MODEL_ELB_NAME", "test-http-model-endpoint")
+    HTTP_CENTERPOINT_MODEL_INFERENCE_PATH = os.getenv("HTTP_CENTERPOINT_MODEL_INFERENCE_PATH", "/invocations")
```
Comment (RanbirAulakh) [inline src/aws/osml/utils/osml_config.py:32]: Are we not doing this for other models? Aircraft and Flood?

Comment (drduhe) [reply src/aws/osml/utils/osml_config.py:32]: SM Abstracts these other params from you - like ELB and the invoke path.

File: src/aws/osml/integ/http_centerpoint/test_http_centerpoint_model.py (hunk 1 above)
Comment (drduhe) [reply src/aws/osml/integ/http_centerpoint/test_http_centerpoint_model.py:59]: Will update

**Review by RanbirAulakh — DISMISSED** [review]

**Review by RanbirAulakh — DISMISSED** [review]

**Review by jtblack-aws — APPROVED** [review]

**Review by apatwal72 — APPROVED** [review]

**Pull request merged**